app/javascript/components/lease_ups/utils/leaseUpRequestUtils.js

```diff
@@ -64,6 +106,9 @@ export const getApplications = async (
 
 export const getListing = async (listingId) => apiService.getLeaseUpListing(listingId)
 
+/**
+ * @deprecated No longer used anywhere.
+ */
```

Comment:
question: are there followup plans for the deprecation comments in this PR? Also, why the `@` sign?

Reply:
The @ sign because it is a magic thing that makes the function crossed out wherever it is used - I wonder if that only works on my linter

Followup plans is a good question. I imagine it should be the last step after the feature flag has been on for a while. @alulabeshue-sfgov maybe we should add a ticket to remove the deprecated code to the current epic?